app/src/test/java/com/example/marvelapp/MainViewModelTest.kt

```diff
@@ -0,0 +1,5 @@
+package com.example.marvelapp
```

Comment:
the ViewModel tests should be inside a ViewModel package

Reply:
done